configure

```diff
@@ -14941,7 +14941,7 @@ fi
                 jxl_lib=`find_lib libjxl.dylib`
                 ;;
             *-*-cygwin* | *-*-mingw*)
-                jxl_lib=`find_lib "libjxl*.dll"`
+                jxl_lib=`find_lib "libjxl(-[0-9]+)?.dll"`
```

Comment:
I don't know regex much but your regex seems correct. However the issue is with actually `find_lib` therefore it doesn't find anything with this.

Reply:
I looked into `find_lib` and I think it is supposed to be a glob, no regex.